draft-ietf-tls-tls13.md

```diff
@@ -1703,42 +1707,62 @@ described below.
-                                                       ServerHello
-                                                        + KeyShare
-                                             + EarlyDataIndication
-                                             {EncryptedExtensions}
-                                            {ServerConfiguration*}
-                                                    {Certificate*}
-                                             {CertificateRequest*}
-                                              {CertificateVerify*}
-                                 <--------              {Finished}
-       {Finished}                -------->
-
-       [Application Data]        <------->      [Application Data]
-
-            () Indicates messages protected using keys
-               derived from the static secret.
+         Client                                               Server
+        
+         ClientHello
+           + KeyShare
+           + EarlyDataIndication
+      ^  (Certificate*)
+0-RTT |  (CertificateVerify*)
+Data  |  (Finished)
+      v (Application Data)         -------->
+                                                         ServerHello
+                                               + EarlyDataIndication
+                                                          + KeyShare
+                                               {EncryptedExtensions}
+                                              {ServerConfiguration*}
+                                               {CertificateRequest*}
+                                                      {Certificate*}
+                                                {CertificateVerify*}
+                                   <--------              {Finished}
+         {Certificate*}
+         {CertificateVerify*}
+         {Finished}                -------->
+        
+         [Application Data]        <------->      [Application Data]
+        
+               *  Indicates optional or situation-dependent
+                  messages that are not always sent.
+   
+               () Indicates messages protected using keys
+                  derived from the static secret.
+   
+               {} Indicates messages protected using keys
+                  derived from the ephemeral secret.
+   
+               [] Indicates messages protected using keys
+                  derived from the master secret.
 ~~~
 {: #tls-0-rtt title="Message flow for a zero round trip handshake"}
 
+As shown in Figure {{tls-0-rtt}}, the Zero-RTT data is just added
+to the 1-RTT handshake in the first flight. Specifically, the
+client sends its Authentication messages after the ClientHello,
+followed by any application data. The rest of the handshake
+messages are the same as with Figure {{tls-full}}. This implies
+that the server can request client authentication even if the
+client offers a certificate on its first flight. This isn't
+necessarily useful but is consistent with the server being
```

Comment:
I think that you might want to avoid statements like "this isn't necessarily useful"